oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -189,6 +189,16 @@ The process for nomination is as follows:
 
 #. During the comment period, other CCs should weigh in on the thread. As much
    as possible, posts should conclude with a definitive "yes" or "no" vote.
+   "Liking" the nomination post does not count as a vote.
+
+   #. In the case of Core Contributor roles that involve granting commit access
+      to repos, anyone who has commit access to the relevant repos may vote;
```

Comment:
In some cases this could be less than 5 votes. In some cases, there may only be people from the same organization. I can't think of anything better to say; I think we should monitor the situation and if _this_ arises, we can deal with it.

Reply:
What if access to multiple repos is being granted? Do all voters need access to all such repos?

And do we give non-commit-related access via the CC program that is worth mentioning here?

Reply:
@kdmccormick 

> What if access to multiple repos is being granted? Do all voters need access to all such repos?

I read this as "at least one of" the repos. Right now, it would be extremely restrictive to say the subset has to match entirely. I'm not sure exactly what to address... do we need to then further specify that there must be not only five overall votes, but at least one affirmative vote for each repo being granted access to?

> And do we give non-commit-related access via the CC program that is worth mentioning here?

I'm not sure what you're referring to. In any case this bullet starts with, `In the case of Core Contributor roles that involve granting commit access`, and I think if other special cases in other roles arise, we can continue to amend this OEP.

Reply:
Sorry, I was reading this on a phone and responded hastily. I also hadn't read the good discussion you and Dave had [up here](https://github.com/openedx/open-edx-proposals/pull/293#discussion_r797956840). Disregard my comment, and let me try again in another thread.

Reply:
Restricting the vote to the people who have access to the repo sends a signal to the other core contributors that their opinion is very secondary. It wouldn't encourage core contributors to look at applicants outside of their own responsibilities, and would imho greatly reduce the number of reviews new applicants would get. And with the current formulation, it even unclear whether a "no" from a core contributor who doesn't have access would count...

I think we need to separate the concerns here - I would suggest:
* All core contributors can vote, regardless of commit rights: a "yes" counts against the 5 minimal votes, and a "no" is a veto vote, regardless of which core contributor posted it
* BUT to get access to a specific repository, at least one of the (positive) votes should come from a core contributor having access to that repository. If there are multiple repositories involved, these votes can come from multiple core contributors having different access.

Reply:
Hey @antoviaque - I am going to resolve this and then copy your comment to the PR itself so that we are all talking in one thread.